### PR TITLE
Update Readme for Github page

### LINKED ISSUE #21: Add more information on README
With more information on README, it would probably be easier to share this course on the Internet. cc @yungyuc

### PR DESCRIPTION
**Pull request by tigercosmos**
resolve #21
Demo: https://tigercosmos.github.io/nsd/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,324 @@ Copyright 2019, Yung-Yu Chen <yyc@solvcon.net>.  All rights reserved.
...
+* W11 (11/23) [Lecture 10](#Lecture-10-Modern-C++): Modern C++ (*homework \#5*)
+* W12 (11/30) [Lecture 11](#Lecture-11-C++-and-C-for-Python): C++ and C for Python
+* W13 (12/7) [Lecture 12](#Lecture-12-Array-code-in-C++): Array code in C++ (*homework \#6*)
+* W14 (12/14) [Lecture 13](#Lecture-13-Array-oriented-design): Array-oriented design
+* W15 (12/21) [Lecture 14](#Lecture-14-Advanced-Python): Advanced Python
+* W16 (12/28) Term project presentation
+* W17 (1/4) No meeting (optional lecture is not planned)
+* W18 (1/11) No meeting (optional lecture is not planned)
+
+# Lecture 1 [Introduction](01_introduction/introduction.ipynb)
```
Comment (yungyuc) [inline README.md:57]: The link link doesn't seem to work?  I think the target should point to `notebook/20au_nctu/01_introduction/01_introduction.ipynb`.

If external links don't work in the GitHub page, we should remove them.

File: README.md [hunk 2]
```diff
@@ -7,3 +7,324 @@ Copyright 2019, Yung-Yu Chen <yyc@solvcon.net>.  All rights reserved.
...
+2. Part 2: What to learn in this course
+   1. Term project
+   2. How to write a proposal
+   3. Term project grading guideline
+   4. Online discussion
+3. Part 3: Runtime and course marterials
+   1. Runtime environment: Linux and AWS
+   2. Jupyter notebook
+
+([bare notebook](01_introduction/introduction_bare.ipynb)) ([schedule toc](#Course-schedule))
```
Comment (yungyuc) [inline README.md:72]: We don't need the links to "bare" files.

**Review by yungyuc — CHANGES_REQUESTED** [review]
Thanks for the addition!  I'd like to confirm the link questions before merging.

File: README.md (hunk 2 above)
Comment (tigercosmos) [reply README.md:72]: Seems anchors don't work in Github Page. (It doesn't create anchors for `H1`, `H2`, but anchors work if viewing in Markdown)
Shall we remove all anchors link to fit Page?

Comment (yungyuc) [reply README.md:72]: OK, let's remove the anchors too.

**Comment by tigercosmos** [comment]
@yungyuc r?

**Comment by yungyuc** [comment]
Looks good!

**Comment by tigercosmos** [comment]
@yungyuc  Remember to turn on the Github Page at setting.

You can add the link at the information part of the Repo.

**Comment by yungyuc** [comment]
Already done: https://yungyuc.github.io/nsd

**Pull request merged**